population-pyramid/parseData.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * General data munging functionality
+ */
+
+import * as d3 from 'd3';
+
+/**
+ * Parses CSV file and returns structured data
+ * @param  {String} url Path to CSV file
+ * @return {Object}     Object containing series names, value extent and raw data object
+ */
+export function fromCSV(url, options) {
```

Comment:
`12:30  error  'options' is defined but never used    no-unused-vars`

Please either remove the `options` arg or add a `// eslint-disable-line no-unused-vars` comment.